generators/python/src/fern_python/generator_cli/readme_snippet_builder.py

```diff
@@ -535,15 +535,18 @@ def _get_default_endpoint_id(
         if ir.readme_config is not None and ir.readme_config.default_endpoint is not None:
             return ir.readme_config.default_endpoint
 
+        # TODO(tjb9dc): The below condition will never match due to non-overlapping endpoint ID types, need to fix
```

Comment:
Gave up on this one because it was really nontrivial for me to get the actual ID